client/src/components/MovieCard/CardInfo/CardInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import { styled } from '@mui/material/styles';
+import { Box, CardContent, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { Movie } from '../../typedefs/typedefs';
+
+const CardInfoStyle = styled(CardContent)(({ theme }) => ({
+  '&:last-child': {
+    paddingBottom: theme.spacing(2),
+  },
+  display: 'flex',
+  justifyContent: 'space-between',
+  flexDirection: 'column',
+  height: '35%',
+}));
+
+interface Props {
+  movie: Movie,
+  isPreviewMode: boolean,
+}
+
+export const CardInfo: FC<Props> = (props) => {
+  const { isPreviewMode, movie } = props;
+
+  return (
+    <CardInfoStyle>
+      <Box>
+        <Typography variant="h6" gutterBottom sx={{ fontSize: 14 }}>
+          {movie.title}
+        </Typography>
+
+        {!isPreviewMode && Array.isArray(movie.genres) ? (
```

Comment:
move this to the separate variable
```suggestion
        {!isPreviewMode && isGenresArray ? (
```